packages/ui/src/components/shared/menu/ExportWalletDialog.tsx

```diff
@@ -12,6 +14,11 @@ import { useWalletState } from 'providers/WalletStateProvider';
 import { TransferableObject } from 'types';
 import { EventName } from 'utils/eventemitter';
 
+enum ExportWalletMethod {
+  QRCode = 'QR Code',
+  JSON = 'JSON File',
```

Comment:
For consistency with the ExportAccount feature, we should remove the `File` word here.
```suggestion
  JSON = 'JSON',
```